src/main/webapp/style.css

```diff
@@ -36,18 +36,22 @@ body {
 .btn {
```

Comment:
Alphabetize this file please.
Should be done for all declarations recursively.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
We will organize/split CSS in a future PR, if that's ok

Reply:
That seems fine. Can you create a tracking bug?

Reply:
Tracking issue #123